pkgs/development/python-modules/langchain-core/default.nix

```diff
@@ -83,6 +83,10 @@ buildPythonPackage rec {
       nix-update --commit --version-regex 'langchain-text-splitters==(.*)' python3Packages.langchain-text-splitters
       nix-update --commit --version-regex 'langchain==(.*)' python3Packages.langchain
       nix-update --commit --version-regex 'langchain-community==(.*)' python3Packages.langchain-community
+      nix-update --commit --version-regex 'langchain-azure-dynamic-sessions==(.*)' python3Packages.langchain-azure-dynamic-sessions
+      nix-update --commit --version-regex 'langchain-huggingface==(.*)' python3Packages.langchain-huggingface
+      nix-update --commit --version-regex 'langchain-mongodb==(.*)' python3Packages.langchain-mongodb
+      nix-update --commit --version-regex 'langchain-openai==(.*)' python3Packages.langchain-openai
```

Comment:
I don't want to add these `partners` libraries to updateScript.
Adding more libraries will make updating more difficult.
https://github.com/langchain-ai/langchain/tree/langchain-core%3D%3D0.2.33/libs

Reply:
Fixed. I removed my changes. Maybe we should tackly this in their own PR.